src/Api/Glances/Glance.php

```diff
@@ -32,17 +32,17 @@
 class Glance
 {
     /**
-     * @var Application Pushover application.
+     * @var Application pushover application
```

Comment:
I need to double check why this is lowercased 🤔

cc @keradus as the first word is the class name it shouldn't be lowercased, right?

Reply:
```
Description of the `phpdoc_annotation_without_dot` rule.

PHPDoc annotation descriptions should not be a sentence.
```

`@var {type} {description}`.
`description`, the `Pushover application.`, is lower-cased and final dot removed.

if you have suggestion how to optimize it, please raise a PR to Fixer

Reply:
Thanks for your feedback, I think this case cannot be handled then, because Pushover is a company name and should always be uppercased. NVM I think it's ok to lowercase it here

Reply:
`the Pushover application` ? :) 
(i know, workaround only)

Reply:
Agreed